### ISSUE TITLE
Unable to see multiple plan outputs

### ISSUE THREAD

**Issue by riemers** [charleszipp/azure-pipelines-tasks-terraform#331]
**Describe the bug**
If i have multiple plan outputs it doesn't show in the tab. It says nothing. If i only have 1 workspace output it works as designed.

**To Reproduce**
Have multiple tasks that produce a 'publishPlanResult' with different names and no output will be shown.

**Expected behavior**
Some sort of dropdown all collection of all changes under the Plan Output Tab

**Screenshots**
![image](https://user-images.githubusercontent.com/682041/221856094-1d8cba2d-79b1-4745-927b-218abd6d7be7.png)
Is empty when using things like:
```
  - ${{ each tfWorkspaceName in parameters.workspaceList }}:
    - task: TerraformCLI@0
      displayName: "Terraform Plan"
      allowTelemetryCollection: false
      inputs:
        backendType: 'azurerm'
        command: 'plan'
        environmentServiceNameAzureRM: $(subscriptionName)
        publishPlanResults: ${{ tfWorkspaceName }}
      env:
        TF_WORKSPACE: ${{ tfWorkspaceName }}
```

**Pipeline Logs**
No output no logs :)

**Agent Configuration**
Latest configs used, latest az cli etc. Default :)

**Additional context**
Might be that this just is not supported, or there needs to be a way to get the terraform output in seperate pages somehow?

**Comment by jmyers82** [comment]
to solve this I think you need to give each plan a unique name.   Can you tack on -<some_random_var> to the end of the plan name.  I am assuming the tfWorkspaceName is unique in this case?  So this should work as you have it?  hmm...

**Comment by raman-nbg** [comment]
Could you solve the issue? If yes, how did you configure the task?

@jmyers82: `-<some_random_var>` is not known to tf.

**Comment by riemers** [comment]
@raman-nbg, i have yet find the time since i am in the middle of another rewrite. But what he means is to add something random to the plan in the pipeline. Presume he means below as example.

```
- task: TerraformCLI@0
  displayName: "Terraform Plan" <- unique name
  inputs:
    provider: 'azurerm'
    command: 'plan'
    workingDirectory: '../terraform' # by default we are in the /s folder.
    environmentServiceNameAzureRM: '${{ parameters.environment.resourceGroupName }}'
```

**Comment by jason-johnson** [comment]
This will be resolved with #339
